aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,259 @@
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Long written format            | {Intl.DateTimeFormat('en-US', { weekday: 'short', month: 'long', day: 'numeric', year: 'numeric' }).format(date)} | When space is available and displaying the day of week is beneficial to the user.                                                                                                                                                                                                      | <Text>{`Intl.DateTimeFormat('en-US', { weekday: 'short',  day: 'numeric',month: 'long', year: 'numeric' }).format(date)`}</Text> |
+| Medium written format          | {Intl.DateTimeFormat('en-US', { dateStyle: 'long' }).format(date)}                                                | When space is available.                                                                                                                                                                                                                                                               | <Text>{`Intl.DateTimeFormat('en-US', { dateStyle: 'long' }).format(date)`}</Text>                                                |
+| Short written format           | {Intl.DateTimeFormat('en-US', { dateStyle: 'medium' }).format(date)}                                              | When space is limited.                                                                                                                                                                                                                                                                 | <Text>{`Intl.DateTimeFormat('en-US', { dateStyle: 'medium' }).format(date)`}</Text>                                              |
+| Numeric locale-specific format | {Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)}                | When the user needs to compare across dates, such as in a column or list. Otherwise, use written format. Numeric format can differ depending on the region, so even if the date is translated by the UI based on the user's locale, numeric dates can still cause confusion for users. | <Text>{`Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)`}</Text>                |
```

Comment:
```suggestion
| Numeric locale-specific format | {Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)}                | When the user needs to compare across dates, such as in a column or list. Otherwise, use written format. <br /> Numeric format can differ depending on the region, so even if the date is translated by the UI based on the user's locale, numeric dates can still cause confusion for users. | <Text>{`Intl.DateTimeFormat('en-US', { month: '2-digit', day: '2-digit', year: 'numeric' }).format(date)`}</Text>                |
```

I'm unclear as to what the third sentence "Numeric format can differ depending on the region, so even if the date is translated by the UI based on the user's locale, numeric dates can still cause confusion for users." is attempting to communicate.

Reply:
I think the intent was to say that written format is a more reliable way to convey a date. The benefits of numeric format are really just for ease of quick comparison. I've removed the third sentence for now, just ended as "Otherwise, use written format."

Reply:
@halocline poor wording on my end. Wanted to convey that numeric formatting is untrustworthy and confusing for users. Say that its 8/12 in US formatting but as a user in a different region you'd expect 15/8 instead. If the system shows 12/8 you might be wondering whether the date format has translated to your locale or if it's using a different region's formatting. Overall written format is more reliable as Taylor mentioned.